app/uk/gov/hmrc/plasticpackagingtax/registration/forms/liability/LiabilityStartDate.scala

```diff
@@ -27,22 +27,15 @@ object LiabilityStartDate {
   val dateFormattingError = "liabilityStartDate.formatting.error"
   val dateOutOfRangeError = "liabilityStartDate.outOfRange.error"
 
-  val dateLowerLimit: LocalDate = LocalDate.of(2022, 4, 1)
-
-  private val dateUpperLimit =
-    if (LocalDate.now().isAfter(dateLowerLimit)) LocalDate.now().plusYears(1)
-    else dateLowerLimit.plusYears(1)
+  val dateLowerLimit: LocalDate = LocalDate.now()
```

Comment:
Are you sure about this change? It looks like 01/04/2022 is the date on which the tax comes into law.

Reply:
I have changed it..